edx_proctoring/templates/proctoring/proctored-exam-status.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="exam-timer">
+<%- gettext("You are taking") %>
+<a href="<%- interpolate(gettext('%(exam_url_path)s'), { exam_url_path: exam_url_path }, true)%>" >
+
+<%- interpolate(gettext(' %(exam_display_name)s '), { exam_display_name: exam_display_name }, true) %>
+</a>
+<%- gettext(" exam as a proctored exam. Good Luck!") %>
```

Comment:
My recommendation for the entire string: "You are taking {Exam_display_name} as a proctored exam. To pass this exam you must submit your responses within the time limit and also pass the online proctoring review."
@partha OK?